FontAtlas.go

```diff
@@ -39,9 +39,14 @@ func (atlas FontAtlas) GlyphRangesJapanese() GlyphRanges {
 	return GlyphRanges(C.iggGetGlyphRangesJapanese(atlas.handle()))
 }
 
-// GlyphRangesChinese describes Default + Japanese + full set of about 21000 CJK Unified Ideographs.
-func (atlas FontAtlas) GlyphRangesChinese() GlyphRanges {
-	return GlyphRanges(C.iggGetGlyphRangesChinese(atlas.handle()))
+// GlyphRangesChineseFull describes Default + Japanese + full set of about 21000 CJK Unified Ideographs.
```

Comment:
Is either of `GlyphRangesChineseFull` or `GlyphRangesChineseSimplifiedCommon` compatible with the previous `GlyphRangesChinese` ?
Because removing that function would demand a bump of major version, which I'd like to avoid. If there is a compatibility, this could be a forwarding function.

Reply:
`GlyphRangesChinese` was renamed to `GlyphRangesChineseFull` in imgui 1.62. I've restored the original function for compatibility reasons, but added `GlyphRangesChineseFull`, and updated `GlyphRangesChinese` to instead call it. 

I wasn't 100% sure on that change, as it introduces an additional function call, but wanted to keep code duplication low and function naming conventions consistent with imgui's.

Reply:
For these circumstances, the forwarding function is the best solution. It avoids a bump of major version, and Go provides options to mark such a function as "deprecated".
Also, the extra function call doesn't hurt. I'm sure there'd be some inlining, and this is an utility function called only once anyway.